core/src/main/java/aima/core/search/basic/informed/AStarSearch.java

```diff
@@ -121,6 +121,10 @@ public AStarSearch(ToDoubleFunction<Node<A, S>> h) {
 		return searchController.solution(node);

```

Comment:
Something similar could be done with searchController.solution(node) being replaced by solution(node). i.e.:

	public List<A> solution(Node<A, S> node) {
 		return searchController.solution(node);
 	}